WordPress/gutenberg
# Add schema support for styles.filter.duotone

## What?

Add styles.filter.duotone to theme.json Schema

Closes #40398

## Why?

This ensures that themes using the Duotone filter property will be able to validate correctly.



## Testing Instructions

1. Create a theme.json with a duotone filter, ex:
```json
{
  "$schema": "https://schemas.wp.org/trunk/theme.json",
  "version": 2,
  "styles": {
    "filter": {
      "duotone": "value"
    }
  }
}
```
2. Use a JSON schema validator to confirm it validates with no errors






## Screenshots or screencast 
N/A
